.ci/ciao_down_setup_env_ubuntu.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
This script is very similar to `.ci/setup_env_ubuntu.sh` which makes me think they could both be refactored to call a common set of functions to avoid duplicated code.